code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm

```diff
@@ -57,8 +57,6 @@
 				charging = 0
 
 /mob/living/simple_animal/hostile/megafauna/legion/death()
-	if(health > 0)
```

Comment:
Because the only way to call this with `health > 0` is wizard mode shenanigans. Or badmin stuff. Making it immune to it makes little sense.

And for Legion, one death is nothing.